bin/generate-docs/index.js

```diff
@@ -12,45 +12,82 @@ const { parse, resolver } = require( 'react-docgen' );
  */
 const { getDescription, getProps, getTitle } = require( './lib/formatting' );
 const {
+	ANALYTICS_FOLDER,
 	PACKAGES_FOLDER,
+	DOCS_FOLDER,
 	deleteExistingDocs,
 	getExportedFileList,
 	getMdFileName,
 	getRealFilePaths,
-	writeTableOfContents,
+	getTocContents,
```

Comment:
Nitpick: TOC Contents is redundant, just sayin'. 😉

Reply:
Good point! Renamed it to `getTocSection`: ab8e53640eb0ba14865c9de59627568bcd279973.